pkg/operator/ceph/object/status.go

```diff
@@ -69,7 +69,10 @@ func updateStatusBucket(client client.Client, name types.NamespacedName, phase c
 		logger.Warningf("failed to retrieve object store %q to update status to %v. %v", name, phase, err)
 		return
 	}
-
+	if objectStore.Status == nil {
+		logger.Warningf("object store %q status is nil", name)
```

Comment:
I don't think we need to log here, let's just initialize.

Reply:
done